src/Microsoft.Identity.Web.OWIN/InternalClaimDetectedException.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Claims;
+
+namespace Microsoft.Identity.Web.OWIN
+{
+    /// <summary>
+    /// The exception that is thrown when an internal ID Token claim used by Microsoft.Identity.Web internally is detected in the user's ID Token.
+    /// </summary>
+    public class InternalClaimDetectedException : Exception
```

Comment:
Everything here needs to be added to the declared API.

Reply:
What Exception form the declared API then do you suggest using that would be appropriate instead of creating a custom exception class?